platforms/documentation/docs/src/snippets/plugins/fileSizeDiffBinaryPlugin/common/plugin/src/main/java/org/example/FileSizeDiffTask.java

```diff
@@ -23,11 +22,6 @@ public abstract class FileSizeDiffTask extends DefaultTask {
     @OutputFile
     public abstract RegularFileProperty getResultFile();
 
-    @Inject
-    public FileSizeDiffTask(ObjectFactory objects) {
-        getResultFile().set(getProject().getLayout().getBuildDirectory().file("diff-result.txt"));
```

Comment:
Instead of configuring this convention in the task itself, it should go up in the plugin